engineering_tools/opendrive_lanelet2_converter/opendrive2lanelet2convertor.py

```diff
@@ -0,0 +1,169 @@
+import os
+import sys, getopt
+from pyproj import Proj, transform
+
+from commonroad.common.file_writer import CommonRoadFileWriter
+from opendrive2lanelet.opendriveparser.parser import parse_opendrive
+from opendrive2lanelet.network import Network
+
+import xml.etree.ElementTree as xml
+import xml.dom.minidom as pxml
+from lxml import etree
+
+# class representing node in lanelet2
+class Node:
+    def __init__(self, id, lat, lon, local_x, local_y):
+        self.id = id
+        self.lat = lat
+        self.lon = lon
+        self.local_x = local_x
+        self.local_y = local_y
+
+    def create_xml_node_object(self):
+        node_element = xml.Element('node',{'id': str(self.id), 'lat': str(self.lat), 'lon': str(self.lon), 'version': str(1), 'visible': 'true'})
+        xml.SubElement(node_element, "tag", {"k": "ele", "v": "0.0"})
+        xml.SubElement(node_element, "tag", {"k": "local_x", "v": str(self.local_x)})
+        xml.SubElement(node_element, "tag", {"k": "local_y", "v": str(self.local_y)})
+        return node_element
+
+# class representing way in lanelet2
+class Way:
+    def __init__(self, id, nodes):
+        self.id = id
+        self.nodes = nodes
+    
+    def create_xml_way_object(self):
+        way_element = xml.Element("way", {"id": self.id, "version": str(1), "visible": "true"})
+        for i in self.nodes:
+            xml.SubElement(way_element, "nd", {"ref": str(i.id)})
+        xml.SubElement(way_element, "tag", {"k": "level", "v": "0"})
+
+        return way_element
+
+# class representing relation in lanelet2
+class Relation:
+    def __init__(self, id, member_left_id, member_right_id, from_cad_id, to_cad_id, cad_id, relation_type):
+        self.id = str(id)
+        self.member_left_id = str(member_left_id)
+        self.member_right_id = str(member_right_id)
+        self.from_cad_id = from_cad_id
+        self.to_cad_id = to_cad_id
+        self.relation_type = relation_type
+        self.cad_id = cad_id
+        self.turn_direction = "straight"
```

Comment:
Add a note to the readme describing how Autoware turn directions are not yet supported.